digitalstrategie/assets/scss/components/_forms.scss

```diff
@@ -0,0 +1,66 @@
+.form--overwrite {
+    & form label {
+        font-weight: normal;
```

Comment:
I think `&` is not needed as it's just a longer way to write it as it's already nested, [here ](https://css-tricks.com/the-sass-ampersand/)is good overview of ampersand. 
Also no need for `form` throughout the whole of the file